modules/alloy/src/main/ts/ephox/alloy/api/events/AlloyTriggers.ts

```diff
@@ -26,15 +25,15 @@ const dispatchWith = (component: AlloyComponent, target: SugarElement, event: st
     target,
     ...properties
   };
-  component.getSystem().triggerEvent(event, target, Obj.map(data, Fun.constant));
+  component.getSystem().triggerEvent(event, target, data);
```

Comment:
Removing the thunk here will make it so the custom event data passed in is no longer thunked.